src/core/Button/Button.md

```diff
@@ -36,11 +36,11 @@ import { Button } from 'suomifi-ui-components';
 
 <>
   <div example="negative">
```

Comment:
This 'negative' was left as it was because in general it indicates just negative version for background colouring for example shown in styleguidist.

It is used in the styleguidist styles:
https://github.com/vrk-kpa/suomifi-ui-components/blob/0876abc22ee2c2b73cb0ca16c2414e4ac0777793/.styleguidist/styleguidist.styles.js#L98

--- 

@LJKaski What are your thoughts about these?
Should we also rename these to `inverted`, even though they are not really the Button-variant? 

Maybe it would make sense, as the `negative` in this case is like the opposite to white background on styleguidist and maybe the `inverted` is more describing than the `negative`?

Reply:
I'd keep the naming consistent and name these inverted as well.

Reply:
This change is now committed.